batcher/aligned-batcher/src/lib.rs

```diff
@@ -48,6 +48,9 @@ pub mod sp1;
 pub mod types;
 mod zk_utils;
 
+#[cfg(test)]
+mod testonly;
```

Comment:
maybe we could name it just `testing`? sounds a little cleaner to me

Reply:
+1